js/attlist.js

```diff
@@ -0,0 +1,103 @@
+var toggle = document.querySelector(".nav-toggle");
+var tog = document.querySelector("HTML");
+
+function toggleModal() {
+    toggle.classList.toggle("active");
+    tog.classList.toggle("openNav")
+}
+
+
+toggle.addEventListener("click", toggleModal);
+
+
+  (function(){
+    /**
```

Comment:
please remove the commented-out code. Why are you using var here instead of const?